src/lib/pages/backoffice/Series/SeriesForms/SeriesEditor/components/SeriesForm/SeriesMetadataExtensions.js

```diff
@@ -0,0 +1,61 @@
+import { extensionsConfig } from '@config';
+import { AccordionField } from '@forms/core/AccordionField';
+import { BooleanField } from '@forms/core/BooleanField';
+import { DateInputField } from '@forms/core/DateTimeFields/DateInputField';
+import { StringField } from '@forms/core/StringField';
+import _merge from 'lodash/merge';
+import _forOwn from 'lodash/forOwn';
+import _get from 'lodash/get';
+import _keys from 'lodash/keys';
+import PropTypes from 'prop-types';
+import React from 'react';
+
+const getFormComponent = fieldType => {
+  switch (fieldType) {
+    case 'string':
+      return StringField;
+    case 'boolean':
+      return BooleanField;
+    case 'date':
+      return DateInputField;
+    default:
+      throw new Error(
+        `${fieldType} did not match any of the supported types: string, date or boolean`
+      );
+  }
+};
+
+export const SeriesMetadataExtensions = ({ extensions }) => {
+  const { label, fields } = extensionsConfig.series;
```

Comment:
Did it make sense to create a re-usable form component for both Document and Series instead?

Reply:
yes it makes, created an issue #69